be/src/exec/vectorized/json_scanner.h

```diff
@@ -102,9 +103,14 @@ class JsonReader {
     int _total_lines;
     bool _closed;
 
-    std::unique_ptr<uint8_t[]> _json_binary_ptr;
+    std::unique_ptr<uint8_t[]> _parser_buffer;
+    size_t _parser_buffer_cap = 0;
+    size_t _parser_buffer_sz = 0;
+
     bool _is_ndjson = false;
 
+    simdjson::ondemand::parser _simdjson_parser;
```

Comment:
Could you provide some comment for this parser? Why do we introduce this parser.